Packs/Legacy/Integrations/IntSight/IntSight.py

```diff
@@ -256,7 +256,8 @@ def alert_to_readable(alert, parse_tags):
         'Description': demisto.get(alert, 'Details.Description'),
         'Title': demisto.get(alert, 'Details.Title'),
         'TakedownStatus': demisto.get(alert, 'TakedownStatus'),
-        'SubType': demisto.get(alert, 'Details.SubType')
+        'SubType': demisto.get(alert, 'Details.SubType'),
+        'Closed': demisto.get(alert, 'Closed')
```

Comment:
the closed field contains a dict?
if so let's extract the data from there, no need for inner dict here